ui/packages/docs/src/components/Hitlag/character_data.json

```diff
@@ -1945,6 +1945,52 @@
       }
     ]
   },
+  "navia": {
+    "attack": [
+      {
+        "ability": "N1",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.06,
+        "canBeDefenseHalt": true,
+        "deployable": false
+      },
+      {
+        "ability": "N2",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.06,
+        "canBeDefenseHalt": true,
+        "deployable": false
+      },
+      {
+        "ability": "N3-1",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.01,
+        "canBeDefenseHalt": false,
+        "deployable": true
+      },
+      {
+        "ability": "N3-2",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.01,
+        "canBeDefenseHalt": false,
+        "deployable": true
+      },
+      {
+        "ability": "N3-3",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.01,
+        "canBeDefenseHalt": false,
+        "deployable": true
+      },
+      {
+        "ability": "N4",
+        "hitHaltTime": 0,
+        "hitHaltTimeScale": 0.06,
+        "canBeDefenseHalt": true,
+        "deployable": false
+      }
```

Comment:
move the value in hitHaltTimeScale to hitHaltTime and set the hitHaltTimeScale to 0.01. this applies to all the entries here